cmd/influxd/run/server.go

```diff
@@ -137,16 +127,17 @@ func NewServer(c *Config, buildInfo *BuildInfo) (*Server, error) {
 		node = influxdb.NewNode(c.Meta.Dir)
 	}
 
+	// In 0.11 we removed MetaServers from node.json.  To avoid confusion for
+	// existing users, force a re-save of the node.json file to remove that property
+	// if it happens to exist.
+	nodeContents, err := ioutil.ReadFile(filepath.Join(c.Meta.Dir, "node.json"))
+	if err == nil && strings.Contains(string(nodeContents), "MetaServers") {
+		node.Save()
+	}
+
```

Comment:
This code was removed in #5957 because it breaks rolling back to `0.10` if necessary.